### ISSUE TITLE
`normal_input` is failing.

### ISSUE THREAD

**Issue by CeresBarros** [bcgov/climr#118]
I think it might be because data downloads are not being triggered by `data_check`, given that `length(c(list_gcm(), list_normal()))` is `>1` even when nothing has been downloaded.

```
❯ checking examples ... ERROR
  Running examples in 'climr-Ex.R' failed
  The error most likely occurred in:
  
  > base::assign(".ptime", proc.time(), pos = "CheckExEnv")
  > ### Name: downscale
  > ### Title: Downscale target rasters to points of interest
  > ### Aliases: downscale
  > 
  > ### ** Examples
  > 
  > 
  > xyz <- data.frame(lon = runif(10, -140, -106), lat = runif(10, 37, 61), elev = runif(10))
  > normal <- normal_input()
  Error: [rast] filename is empty. Provide a valid filename
  Execution halted
  ```

**Comment by kdaust** [comment]
Ah. normal_input is no longer used, as it was meant for the previous non-postgres version. So it should be removed. The replacement is `normal_input_postgis`

**Comment by CeresBarros** [comment]
> Since everything is now on postgis, should we rename the new version to 'normal_input`?

Yes, i think that's a good idea. Will address this ASAP

**Comment by CeresBarros** [comment]
I'm still having issues running `normal_input` -- the problem is happening in the example of `dowscale`.

I've changed the example code to

```
xyz <- data.frame(lon = runif(10, -140, -106), lat = runif(10, 37, 61), elev = runif(10))
thebb <- get_bb(xyz) ##get bounding box based on input points
dbCon <- data_connect() 
normal <- normal_input(dbCon = dbCon, bbox = thebb, cache = TRUE)
gcm_input <- gcm_input(list_gcm()[3], list_ssp()[1], list_period()[2])
downscale(xyz, normal, gcm)
historic <- historic_input()
out <- downscale(xyz, normal, gcm = NULL, historic = historic, ppt_lr = FALSE)
```

yet this is failing:
```
> data_connect()
NULL
Warning message:
In value[[3L]](cond) :
  Error: connection to server at "146.190.244.244", port 5432 failed: Connection refused (0x0000274D/10061)
	Is the server running on that host and accepting TCP/IP connections?

Could not connect to database. Will try using cached data.
```

Could you help me get this example working?

**Comment by kdaust** [comment]
Ah yes - had some server issues yesterday where it crashed a few times. Should be back up and stable now - let me know if it's still being problematic!

**Comment by CeresBarros** [comment]
Seems to be working now :)

**Comment by CeresBarros** [comment]
@kdaust, did was `gcm_input` also deprecated in favour of `gcm_input_postgis`? if so what name do we keep?

**Comment by CeresBarros** [comment]
Similarly, was `list_period` deprecated in favour of `list_gcm_period`?

**Comment by kdaust** [comment]
Yes and yes - I think normal_input and gcm_input are the only ones with old versions. And I changed to list_gcm_period() because we also have historic periods, which is currently list_historic(), but should maybe be chaged to list_historic_period()?

**Comment by CeresBarros** [comment]
fixed in #124
